src/main/java/com/blockout/Pane.java

```diff
@@ -320,14 +322,14 @@ public int getHeight()
      * @param <T>  The type of pane returned
      * @return a Pane of the given ID, if it matches the specified type
      */
-    public final <T extends Pane> T findPaneOfTypeByID(String id, @NotNull Class<T> type)
+    public final <T extends Pane> T findPaneOfTypeByID(final String id, @NotNull final Class<T> type)
     {
-        @Nullable Pane p = findPaneByID(id);
+        final @Nullable Pane p = findPaneByID(id);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AModifiersOrderCheck)